src/components/Register.module.css

```diff
@@ -0,0 +1,64 @@
+.headerBackground {
+  background-image: url(../assets/LAPalmTrees.png);
+  height: 250px;
+  background-size:cover;
+  min-width: 100%;
+}
+
+.card {
+  position: relative;
+}
+
+.h1 {
+  text-align: center;
+}
+
+.checkboxContainer {
```

Comment:
after above changes may not need checkbox css

Reply:
done